pkg/querier/queryrange/roundtrip.go

```diff
@@ -426,3 +448,49 @@ func NewMetricTripperware(
 		return next
 	}, c, nil
 }
+
+// NewInstantMetricTripperware creates a new frontend tripperware responsible for handling metric queries
+func NewInstantMetricTripperware(
+	cfg Config,
+	log log.Logger,
+	limits Limits,
+	schema chunk.SchemaConfig,
+	codec queryrange.Codec,
+	instrumentMetrics *queryrange.InstrumentMiddlewareMetrics,
+	retryMiddlewareMetrics *queryrange.RetryMiddlewareMetrics,
+	shardingMetrics *logql.ShardingMetrics,
+	splitByMetrics *SplitByMetrics,
+) (queryrange.Tripperware, error) {
+	queryRangeMiddleware := []queryrange.Middleware{StatsCollectorMiddleware(), nil}
+
+	//if cfg.CacheResults {
```

Comment:
I think we can delete this for now.  Not sure there's any benefits for caching instant queries later.

Reply:
Almost forgot about this. Thanks!